index.html

```diff
@@ -0,0 +1,136 @@
+    rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+  <header class="container">
+  <!-- start of nav section-->
+  <nav class="nav-wrapper">
+    <!--nav left side-->
+    <div class="nav-left">
+      <a href="#">My Logo</a>
+    </div>
+    <!-- nav right side-->
+    <div class="nav-right">
+
+        <button class="hamburger"><i class="fa fa-bars"></i></button>
+
+    </div>
+  </nav>
+  </header>
+  <!--end of nav -->
+
+  <!--start of  section-->
+    <section class="section-wrapper">
+    <div class="section-header">
+      <h1>I'm Abdullah<br> Glad to see you!</h1>
+    </div>
+    <div class="content-of-section">
+      <p>I'm a software developer!<br> I can help you build a product feature or a website. 
+        look through some of my work and experience if you like what you see and 
+        have a product you needed coded, don't hestiate to contact me.
+      </p>
+    </div>
+
+    <div class="social-icons">
+      <div class="icons-header">
+        <h2>LET'S CONNECT</h2>
+      </div>
+      <ul class="icons-list">
+        <li><a href="#" class="icon-link"><i class="fa fa-twitter"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-github"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-linkedin"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-medium"></i></a></li>
+        <li><a href="#" class="icon-link"><i class="fa fa-angellist"></i></a></li>
+      </ul>
+    </div>
+
+    </section> 
+      <!--work section-->
+ <section class="container-work">
+        <div class="card-info">
+        <div class="card-image">
+          <img src="images/SnapshootPortfolio.png" alt="">
+        </div>
+        <div class="card-header">
+          <h3>Multi-post <br> stories</h3>
+        </div>
+        <ul class="card-details">
+          <li class="card-first-element">CANOPY</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-second-element">Back End Dev</li>
+          <li class="circle-icon"><i class="fa fa-circle"></i></li>
+          <li class="card-third-element">2015</li> 
+          </ul>
+        <div class="card-content">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="btn-button">
+          <li class="btn-languages"><a href="#" class="btn-link">html</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">css</a></li>
+          <li class="btn-languages"><a href="#" class="btn-link">javascript</a></li>
+        </ul>
+        <h3><button class="btn-card">See project</button></h3>
```

Comment:
- [ ] As pointed out by the previous reviewer, please make sure that each card has an image like the original design.
- [ ] Also, please ensure to include the number of cards in the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/164946303-39bd6a96-b740-49ad-8806-28d12b14f5d7.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/164946351-e386185a-f5b0-428e-b517-124a1adf3c29.png)